Overture.podspec

```
@@ -28,6 +28,7 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = "10.9"
   s.tvos.deployment_target = "9.0"
   s.watchos.deployment_target = "2.0"
+  s.swift_version = '5.0'
```

Do you know if this just a minimum version? Or do we need to supply everything, like:
```suggestion
  s.swift_version = '5.0', '5.1', '5.2', '5.3'
```

---

Via an array like RAS? https://github.com/ReactiveCocoa/ReactiveSwift/blob/master/ReactiveSwift.podspec#L24

---

Yeah, though Ruby lets you omit the brackets 🤣

I don't know what the difference is of saying `'5.0'` vs. also listing more recent ones, though. Just wanna make sure we don't break anything!

---

> Yeah, though Ruby lets you omit the brackets 🤣

🤦‍♂️